src/vaishnavi/A21_StringExample3.java

```diff
@@ -0,0 +1,29 @@
+package vaishnavi;
+
+public class A21_StringExample3 {
+
+	int sum = 0;
+	int count = 0;
+	String temp ="";
```

Comment:
if this variable is common for other methods then we should write it at the instance level.